Migrations/CurrencyMigrations.cs

```diff
@@ -0,0 +1,40 @@
+using OrchardCore.Commerce.Indexes;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.Data.Migration;
+
+namespace OrchardCore.Commerce.Migrations
+{
+    public class CurrencyMigrations : DataMigration
+    {
+        IContentDefinitionManager _contentDefinitionManager;
+
+        public CurrencyMigrations(IContentDefinitionManager contentDefinitionManager)
+        {
+            _contentDefinitionManager = contentDefinitionManager;
+        }
+
+        public int Create()
+        {
+            _contentDefinitionManager.AlterTypeDefinition("Currency", builder => builder
+                .WithPart("TitlePart")
```

Comment:
Why do we need the title part?

Reply:
I agree that this is not needed. It was added for better display in the list view. 

If I override the summary view for the currency type the Name property could be shown instead.

Reply:
Name is now stored as DisplayText on the content item. TitlePart is removed.